src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -135,7 +135,7 @@
 	public String aspectRatioVideoTrack;
 	private int videoBitDepth = 8;
 
-	private byte thumb[];
+	private volatile byte[] thumb = null;
```

Comment:
@atamariya The reason for using volatile is simply that I can't persue every such issue I come across while trying to do something else, or the PR will grow huge and I will never finish it nor will it get merged. We've had countless discussions here regarding thread safety before, and I always get "downvoted". I figured that using volatile was an "easy fix" that was neither controveral nor time consuming while it will remedy the situation somewhat. I absolutely think it should be done "properly", although I'd use ```ReentrantReadWriteLock``` to minimize blocking (thumb is typically read more than it is written).